src/index.html

```diff
@@ -4,9 +4,316 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a href ="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo_img">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
+
+          <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+
+          <li class="nav__item"><a href="#smartphones" class="nav__link">Smartphones</a></li>
+
+          <li class="nav__item">
+            <a href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
```

Comment:
please fix the code style
```suggestion
            <a
              href="#laptops&computers"
              class="nav__link"
              data-qa="nav-hover"
            >
```